auth-api/src/auth_api/models/org.py

```diff
@@ -41,6 +41,7 @@ class Org(VersionedModel):  # pylint: disable=too-few-public-methods,too-many-in
     type_code = Column(ForeignKey('org_types.code'), nullable=False)
     status_code = Column(ForeignKey('org_statuses.code'), nullable=False)
     name = Column(String(250), index=True)
+    branch_name = Column(String(250), nullable=True)  # to be used for any additional info as branch name etc
```

Comment:
Cannot have 250 characters for branch name, as the customer_name is limited to 360 characters in CFS. Since customer name will be combination of orgname + branch name, it may exceed the length

Reply:
changed to 100